crates/re_viewport/src/space_view.rs

```diff
@@ -160,10 +135,77 @@ impl SpaceViewBlueprint {
             space_origin,
             queries,
             entities_determined_by_user,
-            auto_properties: Default::default(),
         })
     }
 
+    pub fn save_full(&self, ctx: &ViewerContext<'_>) {
+        let timepoint = TimePoint::timeless();
+
+        let arch = re_types::blueprint::archetypes::SpaceViewBlueprint::new(
+            self.class_identifier().as_str(),
+        )
+        .with_display_name(self.display_name.clone())
+        .with_space_origin(&self.space_origin)
+        .with_entities_determined_by_user(self.entities_determined_by_user)
+        .with_contents(self.queries.iter().map(|q| q.id));
+
+        let mut deltas = vec![];
+
+        if let Ok(row) =
+            DataRow::from_archetype(RowId::new(), timepoint.clone(), self.entity_path(), &arch)
+        {
+            deltas.push(row);
+        }
+
+        for query in &self.queries {
+            add_delta_from_single_component(
+                &mut deltas,
```

Comment:
I suppose it would be a lot nicer to just collect all the components and then call something like `ctx.save_blueprint_components` :thinking:
Don't know if that's hard to do right now, if so don't bother!

Reply:
Queries each go to a different entity-path so they have to be separate rows anyways.

Reply:
ah right then this doesn't make all that much sense here 👍